lib/api/src/js/module.rs

```diff
@@ -254,17 +238,20 @@ impl Module {
     /// validation of the Module.
     pub fn validate(_store: &impl AsStoreRef, binary: &[u8]) -> Result<(), CompileError> {
         let js_bytes = unsafe { Uint8Array::view(binary) };
-        match WebAssembly::validate(&js_bytes.into()) {
-            Ok(true) => Ok(()),
-            _ => Err(CompileError::Validate("Invalid Wasm file".to_owned())),
+        #[allow(unused_unsafe)]
+        unsafe {
```

Comment:
Why we are doing this? `#[allow(unused_unsafe)]` and then `unsafe {`. Doesn't make that much sense.
Perhaps I'm missing something?